gammapy/background/tests/test_fov.py

```diff
@@ -1,2 +1,110 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 from __future__ import print_function, division
+
+from gammapy.background import fill_acceptance_image
```

Comment:
Please always sort imports like this:
- Python stdlib
- Numpy
- scipy
- Astropy
- gammapy